### PR TITLE
add a table to display properties

### PR DESCRIPTION
**Pull request by yiliuTo**
Use a table to introduce aad properties

### CODE REVIEW

File: sdk/spring/azure-spring-boot-starter-active-directory/README.md [hunk 1]
```diff
@@ -47,45 +47,28 @@ The authorization flow for resource server:
...
-* azure.activedirectory.base-uri:    
-Base uri for authorization server, the default value is `https://login.microsoftonline.com/`.
-* azure.activedirectory.authorization-clients:    
-Resource server name that the application is going to visit.
-* azure.activedirectory.authorization-clients.{client-name}.scopes:    
-API permissions of a resource server that the application is going to acquire.
-* azure.activedirectory.authorization-clients.{client-name}.scopes.on-demand:    
-If the authorization flow of current resource server will be executed on demand. The default value is false.
+
+| param                                                        | note                                                         |
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-starter-active-directory/README.md:51]: How about using a more formal way here, to describe params and notes.

File: sdk/spring/azure-spring-boot-starter-active-directory/README.md [hunk 2]
```diff
@@ -47,45 +47,28 @@ The authorization flow for resource server:
...
+| **azure.activedirectory**.jwt-connect-timeout                    | Connection Timeout for the JWKSet Remote URL call.           |
+| **azure.activedirectory**.jwt-read-timeout                       | Read Timeout for the JWKSet Remote URL call.                 |
+| **azure.activedirectory**.jwt-set-cache-lifespan                 | The lifespan of the cached JWK set before it expires, default is 5 minutes. |
+| **azure.activedirectory**.jwt-size-limit                         | Size limit in Bytes of the JWKSet Remote URL call.           |
+| **azure.activedirectory**.post-logout-redirect-uri               | Redirect uri for posting log-out.                            |
+| **azure.activedirectory**.redirect-uri-template                  | Used by the authorization server to return responses containing authorization credentials to the client via the resource owner user-agent. |
+| **azure.activedirectory**.session-stateless                      | If true, the stateless auth filter `AADAppRoleStatelessAuthenticationFilter` will be activated. By default, is value is false, which activates `AADAuthenticationFilter`. |
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-active-directory/README.md:69]: Should we add `Deprecated` here? Because the 2 filter is deprecated.

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: sdk/spring/azure-spring-boot-starter-active-directory/README.md (hunk 2 above)
Comment (yiliuTo) [reply sdk/spring/azure-spring-boot-starter-active-directory/README.md]: How about now?

**Review by chenrujun — APPROVED** [review]
LGTM.

**Pull request merged**